frontend/magic_links/templates/landing.html

```diff
@@ -1,21 +1,70 @@
-{% extends "base.html" %}
-
-{% block content %}
+{%- from "govuk_frontend_jinja/components/button/macro.html" import govukButton
+-%} {% extends "base.html" %} {% block content %}
 
 <div class="govuk-grid-row">
-  <div class="govuk-grid-column-two-thirds">
-    <h1 class="govuk-heading-l">Your applications</h1>
-      <p class="govuk-body">When you select <strong>Continue</strong> we'll check to see if you have started any applications using the email address you visited from. This link will only work once and will expire 24 hours after it was requested.</p>
-    <p class="govuk-body">During your application, if you are inactive for 20 minutes your session will timeout and any information you have not saved will be lost.</p>
-    <p class="govuk-body">You can request a new email link to continue any applications you have in progress.</p>
-    <a href="{{ url_for("api_MagicLinksView_use", link_id=link_id) }}" role="button" draggable="false">
-      <button class="govuk-button" data-module="govuk-button">Continue</button>
-        </a>
-        <h2 class="govuk-heading-m">Get help</h2>
-        <p class="govuk-body">If you have questions about the form or the fund, email us at <a class="govuk-link" href="mailto:COF@levellingup.gov.uk">COF@levellingup.gov.uk</a>.</p>
-        <h2 class="govuk-heading-m">How we'll use your information</h2>
-        <p class="govuk-body">Read how we'll use the information from your application in our <a class="govuk-link" href="https://www.gov.uk/government/publications/community-ownership-fund-privacy-notice">privacy notice</a>.</p>
+  <div class="govuk-grid-column-two-thirds govuk-body">
+    <p class="govuk-caption-l">Community Ownership Fund round 2</p>
+    <h1 class="govuk-heading-xl">
+      Apply for funding to save an asset in your community
+    </h1>
+    <p class="govuk-heading-s">
+      Submission deadline: <span class="govuk-body">midday, {date} 2022</span>
+    </p>
+
+    {{ govukButton({
+      "text": "Continue",
+      "href": url_for('api_MagicLinksView_use', link_id=link_id),
+      "isStartButton": true })
+    }}
+    <h2 class="govuk-heading-m">How to complete your application</h2>
+    <h3 class="govuk-heading-s">Moving through the form</h3>
+    <p>This form is split into sections based on how it will be assessed.</p>
+    <p>You do not have to complete it in order, or all at once.</p>
+    <p>
+      You can preview the
+      <a href="https://fsd-funding-prototype-v2.herokuapp.com/cof-w2-all-questions" class="govuk-link">full list of
+        application questions.</a>
+    </p>
+    <p>
+      If you're inactive for 20 minutes, you'll be signed out and lose any
+      information you have not saved.
```

Comment:
I don't think this is true? We're not using he timeout overlay are we @harryyo?

Reply:
It is soon the prototype 🤔 
https://fsd-funding-prototype-v2.herokuapp.com/s20-application/start-page-english

Reply:
@ramsharma-prog I agree the content is correct as per the prototype, but needs revising as this is not how the application actually behaves I believe CC @harryyo

Reply:
Okay 👍